Header header
int8 controlMode
int8 feedbackWay
int8 workingMode
